tests/scripts/all.sh

```diff
@@ -1332,6 +1332,7 @@ component_test_no_ctr_drbg_classic () {
     scripts/config.py unset MBEDTLS_CTR_DRBG_C
     scripts/config.py unset MBEDTLS_USE_PSA_CRYPTO
     scripts/config.py unset MBEDTLS_SSL_PROTO_TLS1_3
+    scripts/config.py unset MBEDTLS_SSL_RECORD_SIZE_LIMIT
```

Comment:
Instead of doing this, better to add an "#undef MBEDTLS_SSL_RECORD_SIZE_LIMIT" like we have an "#undef MBEDTLS_SSL_EARLY_DATA" in include/mbedtls/config_adjust_ssl.h.